doc/man1/openssl-ca.pod.in

```diff
@@ -156,7 +156,7 @@ Names and values of these options are algorithm-specific.
 
 The password used to encrypt the private key. Since on some
 systems the command line arguments are visible (e.g. Unix with
-the L<ps(1)> utility) this option should be used with caution.
+the L<ps(1)> command) this option should be used with caution.
```

Comment:
I would rephrase that parenthesis, something like "(e.g. when using L<ps(1)> on Unix)"